php/example_code/glue/flight_etl_job_script.py

```diff
@@ -0,0 +1,80 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This script is used by the AWS Glue _getting started with crawlers and jobs_ scenario to
+perform extract, transform, and load (ETL) operations on sample flight data.
+As part of the example, it is uploaded to an Amazon Simple Storage Service (Amazon S3)
+bucket so that AWS Glue can access it.
+"""
+
+# pylint: disable=undefined-variable
+
+# snippet-start:[python.example_code.glue.Scenario_GetStartedCrawlersJobs.job_script]
+import sys
+from awsglue.transforms import *
+from awsglue.utils import getResolvedOptions
+from pyspark.context import SparkContext
+from awsglue.context import GlueContext
+from awsglue.job import Job
+
+"""
+These custom arguments must be passed as Arguments to the StartJobRun request.
```

Comment:
The following values must be passed as Arguments to the StartJobRun request.

Reply:
This is duplicated code from the Python Glue MVP and is Python code, so I'm going to leave it for consistency.